docs/src/administration/users.md

```diff
@@ -72,67 +67,19 @@ Available commands:
 * `platform user:role`
   * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+ Add `user1@example.com` to the project with Project Admin permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link.
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+Say you wanted to give User2 the following roles in different environment types:
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
--  **Viewer** role to the **Production environment**
--  **Contributor** role to **all Staging type environments**
--  **Admin** role to **all Development type environments**
+-  **Viewer** role in the **Production environment**
+-  **Contributor** role in **all staging environments**
+-  **Admin** role in **all development environments**
```

Comment:
Should this still be here? I'm confused by it since it's not connected to an action to take.